src/quo2/components/notifications/activity_log/style.cljs

```diff
@@ -39,11 +39,13 @@
 
 (defn title
   []
-  {:color colors/white})
+  {:color     colors/white
+   :max-width "80%"})
```

Comment:
In the DS, the title is adjusted automatically based on the horizontal space left after showing the timestamp, unread dot and the icon. I think using the percentage is not the way to go.